modules/formic/assets/ships/formicSmall/formicSmall.json

```diff
@@ -0,0 +1,1164 @@
+{
```

Comment:
Whitespace seems off in this file. Maybe bulk-format it? 4 spaces per indentation layer to match the other JSON files including in the same PR :-)